gis/whitebox-tools/Portfile

```diff
@@ -5,114 +5,138 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           cargo 1.0
 
-github.setup        jblindsay whitebox-tools 1.4.0
+github.setup        jblindsay whitebox-tools 2.0.0 v
+revision            0
 categories          gis
 platforms           darwin
 maintainers         {@mholling gmail.com:mdholling} openmaintainer
 description         an advanced geospatial data analysis platform
-long_description    an advanced geospatial data analysis platform
+long_description    ${description}
```

Comment:
Usually you'll want to prefix `${description}` when used this way with the argument expansion operator: `{*}${description}`